lib/ansible/module_utils/facts/system/date_time.py

```diff
@@ -36,7 +35,7 @@ def collect(self, module=None, collected_facts=None):
         # Store the timestamp once, then get local and UTC versions from that
         epoch_ts = time.time()
         now = datetime.datetime.fromtimestamp(epoch_ts)
-        utcnow = utcfromtimestamp(epoch_ts).replace(tzinfo=None)
+        utcnow = datetime.datetime.utcfromtimestamp(epoch_ts)
```

Comment:
This should be updated to not use a deprecated function.

Reply:
Done. Marking as resolved.